packages/sources/kaiko-test/src/endpoint/trades.ts

```diff
@@ -84,7 +84,7 @@ type EndpointTypes = {
   Response: SingleNumberResultResponse
   CustomSettings: typeof customSettings
   Provider: {
-    RequestBody: RequestBody
+    RequestBody: never
```

Comment:
Since `RequestBody` isn't needed, should we also remove the interface above?

Reply:
oh right, i forgot about that